model/telemetry/metrics.yaml

```
@@ -0,0 +1,98 @@
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.sampled_count
+    type: metric
+    metric_name: telemetry.sdk.span.sampled_count
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_size
+    stability: experimental
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_capacity
+    stability: experimental
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_submitted
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_submitted
+    stability: experimental
+    brief: "The number of spans submitted for processing to this span processor."
+    note: |
+      A span is considered submitted when the first callback has been called which triggers processing of the span for the given processor.
+      For the Batching and Simple Span Processors this metric therefore MUST be incremented on invocations of the `on_end` span processor callback.
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_processed
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_processed
+    stability: experimental
+    brief: "The number of spans for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` must be empty. For failed processing, `error.type` must contain the failure cause.
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching Span Processors MUST use `queue full` for spans dropped due to a full queue.
+        examples: ["queue full"]
```

nit: I don't love constants with spaces
```suggestion
        examples: ["queue_full"]
```